ci/docker/entrypoint.sh

```diff
@@ -93,12 +93,6 @@ start_docker() {
 
   local docker_opts="${DOCKER_OPTS:-}"
 
-  # Pass through `--garden-mtu` from gardian container
```

Comment:
what was this doing previously?

Reply:
Setting the mtu value option.  The hardened container doesn't have iproute2 installed so we can't call `ip route get ...`. The docker default mtu value is 1500 bytes which is far lower than the max so I just removed it.